Microsoft.Alm.Authentication/Git/Installation.cs

```diff
@@ -287,15 +280,10 @@ internal bool IsValid()
                 && FileSystem.FileExists(_git);
         }
 
-        public static bool operator ==(Installation install1, Installation install2)
-        {
-            return install1._distribution == install2._distribution
-                && PathComparer.Equals(install1._path, install2._path);
-        }
+        public static bool operator ==(Installation lhs, Installation rhs)
+            => Comparer.Equals(lhs, rhs);
 
-        public static bool operator !=(Installation install1, Installation install2)
-        {
-            return !(install1 == install2);
-        }
+        public static bool operator !=(Installation lhs, Installation rhs)
+            => Comparer.Equals(lhs, rhs);
```

Comment:
Shouldn't this be `!Comparer.Equals(lhs, rhs)`?

Reply:
:+1:

Reply:
Should be good now I guess 😄